### PR TITLE
List MaxCoin (MAX)

### PR DESCRIPTION
**Pull request by ghost**
- Official project URL: https://www.maxcoinproject.org/
- Official block explorer URL: https://explorer.maxcoinproject.net/

### CODE REVIEW

File: src/main/java/bisq/asset/coins/MaxCoin.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.RegexAddressValidator;
```
Comment (blabno) [inline src/main/java/bisq/asset/coins/MaxCoin.java]: Invalid import ordering

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Comment by buckshee** [comment]
Hi, please let me know if the changes for the conflict are valid, it looks like a single line to insert that should be safe to insert, but one further consideration I've probably not done the ideal thing for the commit.

**Comment by blabno** [comment]
@buckshee I have reorganized imports for you to match our code style. Please rebase this PR and squash commits into single one.

**Review by blabno — APPROVED** [review]

**Pull request merged**